packages/e2e/cypress/e2e/run/create-pipelinerun.cy.js

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const namespace = 'e2e';
+describe('Create Simple Pipeline Run', () => {
+  before(() => {
+    cy.exec('kubectl version --client');
+    cy.exec(`kubectl create namespace ${namespace} || true`);
+  });
+  it('should create pipelinerun', { retries: 3 }, function () {
+    const uniqueNumber = `${Math.floor(Math.random() * 100000)}`;
```

Comment:
We could also use `Date.now()` but this is fine, although no need to wrap it in quotes